docs/orchestration/execution/storage_options.md

```diff
@@ -143,6 +143,18 @@ GitLab storage uses a [personal access token](https://docs.gitlab.com/ee/user/pr
 GitLab server users can point the `host` argument to their personal GitLab instance.
 :::
 
+## CodeCommit
```

Comment:
Could you move this to `/orchestration/flow_config/storage.html`? That is a new page where these details are organized

Reply:
Done